### ISSUE TITLE
Latest Gimbal Version

### ISSUE THREAD

**Issue by sb8244** [happydenn/cordova-plugin-gimbal#1]
It looks like this is not compatible with v2 of the SDK. I went ahead and ported it locally for a project, and would be happy to share that. There are several changes to the API that aren't compatible with your code. For instance, there is no success delegation on API setup.

I would recommend packaging the `Gimbal.framework` package for ios if you want to keep it at v1

**Comment by happydenn** [comment]
Hey there! Have been really busy updating the repo. Actually I've got the code for Gimbal v2 SDK ready and it's compatible with both Android and iOS. Since it has an entirely different API so I've made it as a separate plugin. Check out the code at https://github.com/happydenn/cordova-plugin-gimbal2

Sorry haven't update this repo for a while. Maybe I'll add a deprecation notice to the README of this repo.

If you wanna share your findings I'm more than happy to discuss with you on the new codebase. And thanks a bunch for your comment.
